packages/gatsby-transformer-xml/src/__tests__/__snapshots__/gatsby-node.js.snap

```diff
@@ -4,12 +4,8 @@ exports[`Process XML nodes correctly correctly creates nodes from XML 1`] = `
 Array [
   Array [
     Object {
-      "attributes": Object {
-        "id": "bk101",
-      },
       "children": Array [],
-      "content": "",
-      "id": "bk101",
+      "id": "uuid-from-gatsby",
```

Comment:
Seems like this is something that changed. Top level `id` attribute (if it exist) should be used as node's `id`

Reply:
Should be fixed as well.